src/mlpack/methods/ann/loss_functions/cosine_embedding_loss.hpp

```diff
@@ -0,0 +1,156 @@
+/**
+ * @file cosine_embedding_loss.hpp
+ * @author Kartik Dutt
+ *
+ * Definition of the Cosine Embedding loss function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LOSS_FUNCTION_COSINE_EMBEDDING_HPP
+#define MLPACK_METHODS_ANN_LOSS_FUNCTION_COSINE_EMBEDDING_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * Cosine Embeddings Loss function is used for measuring whether two inputs are
+ * similar or dissimilar, using using the cosine distance, and is typically used
```

Comment:
typo: 'using' two times.

Reply:
Ohh sorry about that I will fix this. Thanks.